tools/ssvm-aot/ssvmr.cpp

```diff
@@ -146,12 +130,16 @@ int main(int Argc, const char *Argv[]) {
         break;
       }
     }
+    if (FuncType.Params.size() + 1 < Args.value().size()) {
+      for (size_t I = FuncType.Params.size() + 1; I < Args.value().size();
+           ++I) {
+        const uint64_t Value = std::stoll(Args.value()[I]);
```

Comment:
Why do you set the value type to `uint64_t`?

Reply:
If user give more values than expected FuncType.Params, we cannot determine the value types.
Therefore, set the value types to uint64_t and append to parameter lists and let SSVM logs errors.

Reply:
Okay.